### PR TITLE
Add serizalizers

### LINKED ISSUE #16: Replace jbuilder
Replace jbuilder with some alternative

E. g. https://github.com/rails-api/active_model_serializers

Based on https://github.com/textie/textie/pull/4#discussion_r511653663

### PR DESCRIPTION
**Pull request by DmitryBarskov**
Resolves #16

### CODE REVIEW

File: app/controllers/api/v1/courses/enrollments_controller.rb [hunk 1]
```diff
@@ -8,10 +8,11 @@ class EnrollmentsController < ApplicationController
 
         def show
           if enrollment.present?
-            render :show, status: :ok
+            render json: enrollment, status: :ok
```
Comment (RamilyaNigmatullina) [inline app/controllers/api/v1/courses/enrollments_controller.rb]: What do you think about using?
```suggestion
            respond_with enrollment
```

File: app/controllers/api/v1/courses_controller.rb [hunk 2]
```diff
@@ -6,23 +6,27 @@ class CoursesController < ApplicationController
       expose :courses, -> { Course.all }
       expose :course
 
-      def index; end
+      def index
+        render json: courses
```
Comment (RamilyaNigmatullina) [inline app/controllers/api/v1/courses_controller.rb]: Here too. And in all other places.

File: app/responders/api_responder.rb [hunk 3]
```diff
@@ -1,11 +1,11 @@
 class ApiResponder < ActionController::Responder
   def api_behavior
     if get?
-      display resource
+      render json: resource
```
Comment (RamilyaNigmatullina) [inline app/responders/api_responder.rb]: Why do we need these changes?

File: spec/support/matchers/be_a_datetime_string.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+RSpec::Matchers.define :be_a_datetime_string do |expected_time|
+  match do |actual_string|
+    actual_time = Time.zone.parse(actual_string)
+    return false unless actual_time
+    return true unless expected_time
+
+    (actual_time - expected_time).abs < 1.second
```
Comment (RamilyaNigmatullina) [inline spec/support/matchers/be_a_datetime_string.rb:7]: You can use `timecop` gem if you want. I can show you an example of usage if you want. https://github.com/travisjeffery/timecop

**Review by RamilyaNigmatullina — APPROVED** [review]

File: spec/support/matchers/be_a_datetime_string.rb (hunk 4 above)
Comment (DmitryBarskov) [reply spec/support/matchers/be_a_datetime_string.rb:7]: Yes, I use it, but when some timestamp serialized it is become less _precise_ than one in ActiveRecord model. So when I compare one in a response and in a model I get assertion error.
Let me know if timecop solves this case.

**Pull request merged**